builds/misc/images-release.yaml

```diff
@@ -13,8 +13,6 @@ jobs:
     variables:
       build.registry.address: edgebuilds.azurecr.io
       build.registry.user: EdgeBuilds
-      registry.address: edgerelease.azurecr.io
```

Comment:
Please move these default values in pipeline for all these 4 variables.

Reply:
done